### PR TITLE
Minor tweaks to documentation based on feedback

### PR DESCRIPTION
**Pull request by pankajagrawal16**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/w7J8sa2f](https://trello.com/c/w7J8sa2f)

File: api/shipping-guide_2/api.raml [hunk 1]
```diff
@@ -6,34 +6,34 @@ baseUri: https://api.bring.com/shippingguide
...
+      * [Ability to fetch agreement prices specific to a product on REST](/api/shipping-guide_2/#get-shipment-prices-and-estimated-delivery): Now you can specify multiple products and along with multiple customer number in a single REST request. [How?](/api/shipping-guide_2/#get-shipment-prices-and-estimated-delivery)
+      *  Some improvements to our service offering are planned for 2018-19. More info about upcoming changes will follow to all our partners. Future services will be supported by this new API-version in addition to other improvements. Note that our current API’s will not support future changes in our service offering.
 
       ### How to upgrade
-      Now that Shipping Guide API 2.0 is out you might be wondering how to upgrade to it. The [Upgrade to version 2](/api/shipping-guide_2/upgrade-to-version-2) will help you on your way to version 2 of the Shipping Guide API.
+      Now that Shipping Guide API 2.0 is out you might be wondering how to upgrade to it. The [Upgrade to version 2.0](/api/shipping-guide_2/upgrade-to-version-2) will help you on your way to version 2 of the Shipping Guide API.
```
Comment (anirbansingharoy) [inline api/shipping-guide_2/api.raml]: 2.0 instead of 2 ?

Comment (pankajagrawal16) [reply api/shipping-guide_2/api.raml]: Good catch 👍

File: api/shipping-guide_2/upgrade-to-version-2.html [hunk 2]
```diff
@@ -16,9 +16,9 @@
     </div><!----><div class="layout__item palm-one-whole desk-three-quarters">
       <article class="api">
         <section class="api__section">
-          <h1 class="api__title">Upgrading to version 2</h1>
+          <h1 class="api__title">Upgrading to version 2.0</h1>
           <p align="justify">This is a short summary of the differences between version 1 and 2 of the Shipping Guide API and will help
```
Comment (anirbansingharoy) [inline api/shipping-guide_2/upgrade-to-version-2.html]: 1.x and 2.0 may be ?

**Pull request merged**